packages/importer/src/providers/standardnotes/types.ts

```diff
@@ -0,0 +1,109 @@
+export type ContentTypes = "Note" | "Tag" | "SN|SmartTag" | "SN|Component";
```

Comment:
Instead of a union type here, let's convert this into an `enum`. That `enum` can be used for safe comparision.